index.html

```diff
@@ -64,7 +64,7 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
 
```

Comment:
### Your Lanugage Tag
![image](https://user-images.githubusercontent.com/93039359/195177915-0d7915b3-6998-4da4-bf0a-4898284a4c74.png)

- [ ] Your language tag doesn't have a 4.5:1 color contrast ratio. Kindly make it to follow the requrements and to make your portfolio accessible to color blind users :blush: